docs/src/main/sphinx/release/release-478.md

```diff
@@ -0,0 +1,79 @@
+# Release 478 (22 Oct 2025)
+
+## General
+
+* Add support for column lineage in `UNNEST` clauses. ({issue}`16946`)
+* Add `retry-policy.allowed` configuration property to specify which retry
+  policies can be selected by user. ({issue}`26628`)
+* Add support for loading plugins from multiple directories. ({issue}`26855`)
+* Add the `/v1/integrations/gateway` endpoint for integration with Trino Gateway. ({issue}`26548`)
+* Allow to drop an uninitialized catalog that failed to load. ({issue}`26918`)
+* Improve performance of queries with an `ORDER BY` clause. ({issue}`26725`)
+* Improve performance of `MERGE` statements involving a `NOT MATCHED` case. ({issue}`26759`)
+* Improve performance of spilling join queries. ({issue}`26076`)
```

Comment:
Should we say something more explicit like "queries with joins which spill to disk"?